packages/react-ui-components/src/TextArea/textArea.js

```diff
@@ -102,4 +140,9 @@ class TextArea extends PureComponent {
     }
 }
 
-export default TextArea;
+//
+// Add the click-outside functionality to the TextArea component.
+//
+const EnhancedTextArea = enhanceWithClickOutside(TextArea);
```

Comment:
My bet is that the tests are failing because of this line. The tests shallow-render the top component, and in this case it's a component returned by `enhanceWithClickOutside`, not TextArea itself.
To quickly fix the tests you'd need to unwrap that component in tests.